### PR TITLE
Fix replication activity order reversing on action click

### PR DESCRIPTION
**Pull request by popojargo**
## Overview

Fix reorder on action click in replication activity view.

## Testing recommendations

grunt nightwatch --file=replicationactivity

## Steps to reproduce

1. Start a clean CouchDB instance without no _replicator documents
2. Login
3. Go to the _replicator view
4. Add a One time replicaton on "fake" databases
5. Add another One time replication on "fake" databases
6. Sort by type
7. Click the delete button
8. The order should have change 👎 

## GitHub issue number

#973 

## Checklist

- [x] Code is written and works correctly;
- [x] Changes are covered by tests;
- [ ] Documentation reflects the changes;
- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the correct tag once a new Fauxton release is made

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ module.exports = function (grunt) {
 
     exec: {
       start_nightWatch: {
-        command: __dirname + '/node_modules/nightwatch/bin/nightwatch' +
+        command: 'node ' + __dirname + '/node_modules/nightwatch/bin/nightwatch' +
```
Comment (garrensmith) [inline Gruntfile.js]: Is this a windows fix, or why do we need this change?

**Comment by garrensmith** [comment]
This must be a IE issue. I can't reproduce it in Chrome. Could you give me the steps to create the error. The code looks good though.

File: Gruntfile.js (hunk 1 above)
Comment (popojargo) [reply Gruntfile.js]: Without this fix, I can't run grunt nightwatch on Windows.

**Comment by popojargo** [comment]
@garrensmith  Steps  added

**Comment by Antonio-Maranhao** [comment]
+1

**Pull request merged**